common/src/main/java/org/openmrs/analytics/JdbcConnectionUtil.java

```diff
@@ -0,0 +1,71 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.openmrs.analytics;
+
+import java.beans.PropertyVetoException;
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+import com.mchange.v2.c3p0.ComboPooledDataSource;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class JdbcConnectionUtil {
+	
+	private static final Logger log = LoggerFactory.getLogger(JdbcConnectionUtil.class);
+	
+	private static ComboPooledDataSource comboPooledDataSource;
+	
+	private String jdbcDriverClass;
+	
+	private String jdbcUrl;
+	
+	private String dbUser;
+	
+	private String dbPassword;
+	
+	private Integer initialPoolSize;
+	
+	private Integer dbcMaxPoolSize;
+	
+	JdbcConnectionUtil(String jdbcDriverClass, String jdbcUrl, String dbUser, String dbPassword, int dbcMaxPoolSize,
```

Comment:
utilities are not intended to be instantiated. Rather provide static methods for generic functionality